ratis-server/src/test/java/org/apache/ratis/server/impl/GroupManagementBaseTest.java

```diff
@@ -255,4 +258,49 @@ public void testGroupAlreadyExists() throws Exception {
       cluster.shutdown();
     }
   }
+  
+  @Test
+  public void testGroupRemove() throws Exception {
```

Comment:
I see an exception while running the test.
```
2020-07-30 16:00:49,922 [grpc-default-executor-0] TRACE util.FileUtils (FileUtils.java:moveDirectory(97)) - moveDirectory source: /Users/ljain/codebase/apache/incubator-ratis-2/ratis-test/target/test/data/e87f4236/MiniRaftClusterWithGrpc77d65499/s0/ee7ea209-68e0-4fbd-97ce-f85fd0542aee dest: /private/var/folders/2j/h06ldsm970b_jyqskbqwz49m0000gn/T/groups5576348764218221965/ee7ea209-68e0-4fbd-97ce-f85fd0542aee
2020-07-30 16:00:49,922 [grpc-default-executor-0] TRACE util.FileUtils (LogUtils.java:runAndLog(49)) - Successfully ran Files.createDirectories /private/var/folders/2j/h06ldsm970b_jyqskbqwz49m0000gn/T/groups5576348764218221965/ee7ea209-68e0-4fbd-97ce-f85fd0542aee
2020-07-30 16:00:49,926 [grpc-default-executor-0] TRACE util.FileUtils (LogUtils.java:runAndLog(41)) - Failed to Files.move /Users/ljain/codebase/apache/incubator-ratis-2/ratis-test/target/test/data/e87f4236/MiniRaftClusterWithGrpc77d65499/s0/ee7ea209-68e0-4fbd-97ce-f85fd0542aee/current/raft-meta to /private/var/folders/2j/h06ldsm970b_jyqskbqwz49m0000gn/T/groups5576348764218221965/ee7ea209-68e0-4fbd-97ce-f85fd0542aee/current/raft-meta
java.nio.file.NoSuchFileException: /Users/ljain/codebase/apache/incubator-ratis-2/ratis-test/target/test/data/e87f4236/MiniRaftClusterWithGrpc77d65499/s0/ee7ea209-68e0-4fbd-97ce-f85fd0542aee/current/raft-meta -> /private/var/folders/2j/h06ldsm970b_jyqskbqwz49m0000gn/T/groups5576348764218221965/ee7ea209-68e0-4fbd-97ce-f85fd0542aee/current/raft-meta
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixCopyFile.move(UnixCopyFile.java:457)
	at sun.nio.fs.UnixFileSystemProvider.move(UnixFileSystemProvider.java:262)
	at java.nio.file.Files.move(Files.java:1395)
	at org.apache.ratis.util.FileUtils.lambda$move$6(FileUtils.java:79)
	at org.apache.ratis.util.LogUtils.runAndLog(LogUtils.java:38)
	at org.apache.ratis.util.FileUtils.move(FileUtils.java:78)
	at org.apache.ratis.util.FileUtils$1.visitFile(FileUtils.java:114)
	at org.apache.ratis.util.FileUtils$1.visitFile(FileUtils.java:100)
	at java.nio.file.Files.walkFileTree(Files.java:2670)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at org.apache.ratis.util.FileUtils.moveDirectory(FileUtils.java:100)
	at org.apache.ratis.server.impl.RaftServerImpl.groupRemove(RaftServerImpl.java:284)
	at org.apache.ratis.server.impl.RaftServerProxy.lambda$groupRemoveAsync$12(RaftServerProxy.java:418)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:602)
	at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:614)
	at java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:1983)
	at org.apache.ratis.server.impl.RaftServerProxy.groupRemoveAsync(RaftServerProxy.java:416)
	at org.apache.ratis.server.impl.RaftServerProxy.groupManagementAsync(RaftServerProxy.java:372)
	at org.apache.ratis.grpc.server.GrpcAdminProtocolService.lambda$groupManagement$0(GrpcAdminProtocolService.java:43)
	at org.apache.ratis.grpc.GrpcUtil.asyncCall(GrpcUtil.java:165)
	at org.apache.ratis.grpc.server.GrpcAdminProtocolService.groupManagement(GrpcAdminProtocolService.java:43)
	at org.apache.ratis.proto.grpc.AdminProtocolServiceGrpc$MethodHandlers.invoke(AdminProtocolServiceGrpc.java:367)
	at org.apache.ratis.thirdparty.io.grpc.stub.ServerCalls$UnaryServerCallHandler$UnaryServerCallListener.onHalfClose(ServerCalls.java:172)
	at org.apache.ratis.thirdparty.io.grpc.internal.ServerCallImpl$ServerStreamListenerImpl.halfClosed(ServerCallImpl.java:331)
	at org.apache.ratis.thirdparty.io.grpc.internal.ServerImpl$JumpToApplicationThreadServerStreamListener$1HalfClosed.runInContext(ServerImpl.java:818)
	at org.apache.ratis.thirdparty.io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)
	at org.apache.ratis.thirdparty.io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:123)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
2020-07-30 16:00:49,928 [grpc-default-executor-0] INFO  util.FileUtils (FileUtils.java:visitFile(116)) - Files.moveDirectory: could not delete raft-meta
```

Reply:
I couldn't recreate this error. Can you please test again.

Reply:
I also cannot reproduce the error.